src/Schema/Blueprint.php

```diff
@@ -18,19 +20,19 @@
 
 class Blueprint extends SchemaBlueprint
 {
+    private static bool $hasConnectionInConstructor;
+
     /**
-     * The MongoConnection object for this blueprint.
-     *
-     * @var Connection
+     * The MongoDB connection object for this blueprint.
      */
-    protected $connection;
+    protected Connection $connection;
```

Comment:
Type added in Laravel 12.

Reply:
May be worth adding a comment in the doc block above to explain this. Just a note that Laravel 12 added the type hint and that necessitated doing so here.